css-brochure-form/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h3>LEARN MORE</h3>
+  <form action="">
+    <label for="form">COMPANY EMAIL</label>
+    <input type="text" placeholder="sally@company.co">
+    <label for="form">NAME</label>
+    <input type="text" placeholder="Sally Smith">
+    <label for="form">WHERE ARE YOU HIRING?</label>
+    <input type="text" placeholder="City, Country">
+    <label for="form">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
+    <select name="form" id="selectValue">
+      <option value="">1-2 Engineers</option>
+    </select>
```

Comment:
Your labels are not properly configured to pair with the form control element that they are labeling.